airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_assets.py

```diff
@@ -256,7 +256,7 @@ def test_should_respond_200(self, test_client, session):
                     "extra": {"foo": "bar"},
                     "created_at": tz_datetime_format,
                     "updated_at": tz_datetime_format,
-                    "consuming_dags": [],
+                    "scheduled_dags": [],
```

Comment:
We should apply this to the entire file
```suggestion
                    "scheduled_dags": [],
                    "consuming_tasks": [], 
```